charts/sentry/values.yaml

```diff
@@ -1631,7 +1639,7 @@ nginx:
   #   name: nginx
 
 ingress:
-  enabled: false
+  enabled: true
```

Comment:
Why did you decide to turn on ingress by default?

Reply:
Thank you for your question! I decided to enable Ingress by default to streamline the deployment process for most users. Enabling Ingress out-of-the-box simplifies the configuration for environments where external access to the services is required, which is a common scenario.

Reply:
@TartanLeGrand what's the value of enabling ingress though without pairing it with configuring a hostname? New users will need to configure a working hostname for enabling ingress to do anything, there's no way to make ingress work out-of-the-box with zero configuration, so flipping `enabled` to true might as well accompany configuring a hostname so that invalid manifests aren't produced by default